src/org/openstreetmap/josm/gui/tagging/presets/TaggingPresetSelector.java

```diff
@@ -64,13 +63,12 @@ public class TaggingPresetSelector extends SearchTextResultListPanel<TaggingPres
 
     private static final Pattern PATTERN_PUNCTUATION = Pattern.compile("\\p{Punct}", Pattern.UNICODE_CHARACTER_CLASS);
 
-    private static final BooleanProperty SEARCH_IN_TAGS = new BooleanProperty("taggingpreset.dialog.search-in-tags", true);
-    private static final BooleanProperty ONLY_APPLICABLE = new BooleanProperty("taggingpreset.dialog.only-applicable-to-selection", true);
-    private static final BooleanProperty DISPLAY_DEPRECATED = new BooleanProperty("taggingpreset.dialog.display-deprecated", true);
-
-    private final JCheckBox ckOnlyApplicable;
-    private final JCheckBox ckSearchInTags;
-    private final JCheckBox ckDeprecated;
+    PresetSearchFilter ONLY_APPLICABLE = PresetSearchFilter.ONLY_APPLICABLE;
+    PresetSearchFilter SEARCH_IN_TAGS = PresetSearchFilter.SEARCH_IN_TAGS;
+    PresetSearchFilter DEPRECATED_TAGS = PresetSearchFilter.DEPRECATED_TAGS;
```

Comment:
You shouldn't need/want to do this. Importing them statically would work, or you can just use the qualified name (`PresetSearchFilter.DEPREACATED_TAGS`). I only saw these used in a `switch` statement, where they aren't needed.